packages/ember-glimmer/tests/integration/components/class-bindings-test.js

```diff
@@ -45,6 +45,58 @@ moduleFor('ClassNameBindings integration', class extends RenderingTest {
     this.assertComponentElement(this.firstChild, { tagName: 'div', attrs: { 'class': classes('ember-view foo enabled sad') }, content: 'hello' });
   }
 
+  ['@test it can have class name bindings in the template']() {
+    this.registerComponent('foo-bar', { template: 'hello' });
+
+    this.render('{{foo-bar classNameBindings="someInitiallyTrueProperty someInitiallyFalseProperty someInitiallyUndefinedProperty :static isBig:big isOpen:open:closed isUp::down"}}', {
```

Comment:
in order for this to follow the INUR pattern, you need have one more nested level that you can interior update then replace